src/components/ClusterManager.js

```diff
@@ -167,7 +167,7 @@ export default class ClusterManager extends PureComponent {
       setDynamic(sessionStorage, `notifiedOutdatedCluster${cluster.id}`, Date.now())
       notify('warn', 'Please Update Your Runtime', {
         message: h(Fragment, [
-          p(['On Sunday Oct 20th at 10am, we are introducing important updates to Terra, which are not compatible with the older notebook runtime in this workspace. After this date, you will no longer be able to save new changes to notebooks in one of these older runtimes.']),
```

Comment:
Should this date include the year and time zone? i.e. 'Sunday Oct 20th, 2020 at 10am EST' ? (Also if this time is based on the TerraUI deployment cycle we actually release at 11am now (due to daylights savings time switch and we didn't adjust it))

Reply:
I don't think it needs to be quite that specific, but I can update to "Last year in October" if you think that would be helpful to users.

Reply:
I think @ehanna4 was looking at the old removed text and not the replacement...?

Reply:
🤦‍♀️yup sorry, never mind on my last comment